dazel/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.3.2
+
+* Add suggestions for .gitignore
```

Comment:
Thanks for doing this, we should try to remember to push back on reviews that don't include changelog updates (most of these were my fault)